### PR TITLE
add dataset name in swds input.json

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
- add dataset name in swds input.json
## Modules
- [ ] UI
- [x] Controller
- [x] Agent
- [ ] Client
- [x] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [x] add necessary doc

### CODE REVIEW

File: client/starwhale/api/_impl/loader.py [hunk 1]
```diff
@@ -92,12 +92,12 @@ def __init__(self, storage: StorageBackend, swds: list=[], logger: t.Union[logur
     def __iter__(self):
         for _swds in self.swds:
             for data, label in zip(
-                self._do_iter(_swds["bucket"], _swds["key"]["data"]),
-                self._do_iter(_swds["bucket"], _swds["key"]["label"])
+                self._do_iter(_swds["bucket"], _swds["key"]["data"], _swds["ext_attr"]),
```
Comment (tianweidut) [inline client/starwhale/api/_impl/loader.py:95]: `_swds.get('ext_attr', {})`

File: client/starwhale/api/_impl/model.py [hunk 2]
```diff
@@ -214,7 +214,8 @@ def _starwhale_internal_run_ppl(self) -> None:
                     data.data, data.batch_size,
                     data_index=data.index, data_size=data.data_size,
                     label_content=label.data, label_size=label.data_size,
-                    label_batch=label.batch_size, label_index=label.index)
+                    label_batch=label.batch_size, label_index=label.index,
+                    ds_name=data.ext_attr["ds_name"], ds_version=data.ext_attr["ds_version"])
```
Comment (tianweidut) [inline client/starwhale/api/_impl/model.py:218]: `data.ext_attr.get('ds_name', "")`

**Review by tianweidut — CHANGES_REQUESTED** [review]
others lgtm

**Review by goldenxinxing — APPROVED** [review]
lgtm

**Review by goldenxinxing — APPROVED** [review]
lgtm

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**